src/harness/testcases/WINNF_FT_S_SIQ_testcase.py

```diff
@@ -231,7 +231,12 @@ def test_WINNF_FT_S_SIQ_3(self):
 
     # Checks for cbsd 1 (index 0)
     self.assertChannelsContainFrequencyRange(channels_low[0], freq_range_low)
-    self.assertChannelsContainFrequencyRange(channels_high[0], freq_range_high)
+    constrain_low = True
+    constrain_high = False
+    self.assertChannelsOverlapFrequencyRange(channels_high[0], freq_range_high, constrain_low, constrain_high)
```

Comment:
You can just do:

```
self.assertChannelsOverlapFrequencyRange(channels_high[0], freq_range_high, constrain_low=True, constrain_high=False)
```

This is how you use keyword args in Python, and it makes the code more compact and readable.